lib/rpmsg/rpmsg.c

```diff
@@ -251,57 +251,45 @@ void rpmsg_destroy_ept(struct rpmsg_endpoint *ept)
  * @return - status of function execution
  */
 
-int rpmsg_init_vdev(struct rpmsg_virtio_device *rvdev,
+int rpmsg_init_master_vdev(struct rpmsg_virtio_device *rvdev,
 		    struct virtio_device *vdev, struct metal_io_region *shm_io,
 		    void *shm, unsigned int len)
 {
 	const char *vq_names[RPMSG_NUM_VRINGS];
 	void (*callback[RPMSG_NUM_VRINGS]) (struct virtqueue *vq);
 	unsigned long dev_features;
 	static struct rpmsg_endpoint ns_ept;
+	struct virtqueue_buf vqbuf;
+	void * buffer;
 	int status;
-	unsigned int i;
+	unsigned int i, idx;
```

Comment:
how about just "i"?

Reply:
yes,  no sense to re-declare a local variable, i will update this